### DISCUSSION TITLE
Which organizations should be involved?

### DISCUSSION THREAD

**Discussion by kinlane** [OAI/sig-travel#5] [General]
What organizations should this group be approaching, inviting, and working with as part of our discussions?

**Comment by swaldron58** [comment]
HTNG, now part of AHLA. 
OSDF, a joint effort by the UIC and FSM to define how existing rail ticketing functions can be publish as JSON. 
Open Air API, an effort by IATA, similar to the rail effort above.

**Reply by kinlane** [reply]
Hospitality Technology Next Generation (HTNG) - https://www.htng.org/

**Reply by kinlane** [reply]
International Union of Railways (UIC) - https://uic.org/

**Reply by kinlane** [reply]
Full Service Model (FSM) - http://www.cer.be/full-service-model-fsm

**Reply by kinlane** [reply]
Open Sales and Distribution Model (OSDM) - https://unioninternationalcheminsdefer.github.io/OSDM/

**Reply by kinlane** [reply]
IATA - https://www.iata.org/

**Reply by kinlane** [reply]
IATA Open Air - https://developer.iata.org/en/api/

**Reply by kinlane** [reply]
OpenTravel - https://opentravel.org/

**Reply by Keith-Heslop** [reply]
Rail Journey Information System - [https://www.fujitsu.com/uk/Images/atoc.pdf[](url)](url) and [https://publications.parliament.uk/pa/cm200304/cmselect/cmtran/580/580we02.htm](url)
